### PR TITLE
NO-JIRA: tls registry: don't store annotation values in violations

### PR DESCRIPTION
**Pull request by vrutkovs**
Violations JSON should only contain artifact locations and should not
be updated when their annotation changes

### CODE REVIEW

**Review by dinhxuanvu — APPROVED** [review]
/lgtm

File: vendor/github.com/openshift/library-go/pkg/certs/cert-inspection/certgraphapi/type_registry.go [hunk 1]
```diff
@@ -42,7 +42,7 @@ type PKIRegistryInClusterCABundle struct {
 	// ConfigMapLocation points to the configmap location
 	ConfigMapLocation InClusterConfigMapLocation `json:"configMapLocation"`
 	// CABundleInfo stores metadata for the certificate authority bundle
-	CABundleInfo PKIRegistryCertificateAuthorityInfo `json:"certificateAuthorityBundleInfo"`
+	CABundleInfo PKIRegistryCertificateAuthorityInfo `json:"certificateAuthorityBundleInfo,omitempty"`
```
[the remark this answers is not in the document]
Comment (vrutkovs) [reply vendor/github.com/openshift/library-go/pkg/certs/cert-inspection/certgraphapi/type_registry.go:45]: Oops, we don't need that part

**Review by dinhxuanvu — COMMENTED** [review]
/lgtm

**Pull request merged**